### ISSUE TITLE
Home - Photo Carousel Tabbing Navigation

### ISSUE THREAD

**Issue by badtigra** [arifire21/lei-site#11]
Have clearer indications that arrows & individual images are selected via tabbing navigation. Depending on image background, it can make it difficult to see what's selected or not.

![Screenshot 2024-06-15 092217](https://github.com/arifire21/Loucks-Engineering-Site/assets/81314819/0aa33bb8-536e-4eb4-9bf7-eb199f62afe6)

**Comment by arifire21** [comment]
@badtigra verify if this is still an issue with the new carousel (different library, different arrow indicator style)

**Comment by arifire21** [comment]
@badtigra also verify arrow styles are okay

**Comment by badtigra** [comment]
> [@badtigra](https://github.com/badtigra) also verify arrow styles are okay

Not an accessibility issue, but I would recommend making the arrow fit more within the bubble.
<img width="58" height="37" alt="Image" src="https://github.com/user-attachments/assets/0d8cf9f3-9679-4cf6-8417-984262d05e43" />

Further, the bar navigation is hard to see with some images. I cannot focus the carousel via keyboard either, but if focused via mouse click it works with arrow keys.

<img width="602" height="60" alt="Image" src="https://github.com/user-attachments/assets/e5e590c4-1b2d-413d-83cb-d5150abe5840" />
